client/state/jetpack-connect/reducer/jetpack-connect-selected-plans.js

```diff
@@ -2,19 +2,11 @@
 /**
  * Internal dependencies
  */
-import {
-	JETPACK_CONNECT_CHECK_URL,
-	JETPACK_CONNECT_COMPLETE_FLOW,
-	JETPACK_CONNECT_SELECT_PLAN_IN_ADVANCE,
-} from 'state/action-types';
+import { JETPACK_CONNECT_CHECK_URL, JETPACK_CONNECT_COMPLETE_FLOW } from 'state/action-types';
 import { jetpackConnectSelectedPlansSchema } from './schema';
-import { urlToSlug } from 'lib/url';
 
 export default function jetpackConnectSelectedPlans( state = {}, action ) {
```

Comment:
It appears to me that we can remove this entire part of the state tree now. It doesn't seem to do anything meaninful.